src/Compilers/CSharp/Portable/Symbols/ConstraintsHelper.cs

```diff
@@ -528,7 +528,9 @@ private static bool CheckConstraintsSingleType(TypeSymbol type, CheckConstraints
             SyntaxNode typeSyntax,
             ImmutableArray<Location> elementLocations,
             Compilation currentCompilation,
-            DiagnosticBag diagnostics)
+            DiagnosticBag diagnosticsOpt,
```

Comment:
>diagnosticsOpt [](start = 26, length = 14)

Are there callers that pass `null`?

Reply:
Yes. My change in `NullableWalker` explicitly does this. 

---
In reply to: [257400295](https://github.com/dotnet/roslyn/pull/33336#discussion_r257400295) [](ancestors = 257400295)